Twitch/Twitch.Base/Twitch.Base.csproj

```diff
@@ -27,6 +27,7 @@ For more information, please see our project site: https://github.com/SaviorXTan
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
```

Comment:
What is this package in reference to specifically?

Reply:
That was so I could use the dynamic type to have type resolution done at runtime for the returned data type.  This allows any model to be passed into the GetPagedDataSegmentsResultAsync method that has a list of some type with the property name of segments.   It was a way to try to abstract this out as much as I could so the method could be reused with other data sets that might return a single data object that has a list property where paged data is stored.

Reply:
I'd like to avoid the use of dynamics if we can, it makes it more difficult to trace the code as well as determine when a breaking change could occur.